helmfile/overrides/notify/admin.yaml.gotmpl

```diff
@@ -19,10 +19,13 @@ admin:
   HC_FR_SERVICE_ID: "{{ .StateValues.HC_FR_SERVICE_ID }}"
   REDIS_ENABLED: "{{ .StateValues.REDIS_ENABLED }}"
   SHOW_STYLE_GUIDE: "{{ .StateValues.SHOW_STYLEGUIDE }}"
+  NEW_RELIC_ENABLED: "{{ .StateValues.NEW_RELIC_ENABLED }}"
   NEW_RELIC_APP_NAME: "notification-admin-{{ .Environment.Name }}"
-  NEW_RELIC_DISTRIBUTED_TRACING_ENABLED: "{{ .StateValues.NEW_RELIC_DISTRIBUTED_TRACING_ENABLED }}"
-  NEW_RELIC_MONITOR_MODE: "{{ .StateValues.NEW_RELIC_MONITOR_MODE }}"
-  NEW_RELIC_IGNORE_STARTUP_ERRORS: "{{ .StateValues.NEW_RELIC_IGNORE_STARTUP_ERRORS }}"
+  NEW_RELIC_DISTRIBUTED_TRACING_ENABLED: {{ if eq .StateValues.NEW_RELIC_ENABLED "true" }}"{{ .StateValues.NEW_RELIC_DISTRIBUTED_TRACING_ENABLED }}"{{ else }}"false"{{ end }}
+  # THESE NEXT TWO MIRROR EACH OTHER AT ALL TIMES
+  NEW_RELIC_MONITOR_MODE: {{ if eq .StateValues.NEW_RELIC_ENABLED "true" }}"{{ .StateValues.NEW_RELIC_MONITOR_MODE }}"{{ else }}"false"{{ end }}
+  NEW_RELIC_EXTENSION_SEND_FUNCTION_LOGS: {{ if eq .StateValues.NEW_RELIC_ENABLED "true" }}"{{ .StateValues.NEW_RELIC_MONITOR_MODE }}"{{ else }}"false"{{ end }}
+  NEW_RELIC_IGNORE_STARTUP_ERRORS: {{ if eq .StateValues.NEW_RELIC_ENABLED "true" }}"{{ .StateValues.NEW_RELIC_IGNORE_STARTUP_ERRORS }}"{{ else }}"false"{{ end }}
```

Comment:
The `NEW_RELIC_IGNORE_STARTUP_ERRORS` var env should always be true, except in a local dev setup where developers can override this in their own environment.